Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN apt-get update && apt-get install -y \
   uuid-runtime \
   wget
 
-ARG CEPH_REPO_URL=https://download.ceph.com/debian-luminous/
+ARG CEPH_REPO_URL=https://download.ceph.com/debian-nautilus/
```

Comment:
This is fine but would maybe have made sense in a separate PR just to make running tests by hand use nautilus by default.

Reply:
this is in a separate commit (https://github.com/ceph/go-ceph/pull/111/commits/df05dc56d56badf92e495e8f50e1e656841763b6). If you prefer to have it in its own PR, I'll move it there.

Reply:
Not today. Just something to consider in the future.